src/lib/components/Sidebar/SidebarItem.tsx

```diff
@@ -31,32 +88,8 @@ const SidebarItem = forwardRef<Element, SidebarItemProps>(
     const { isInsideCollapse } = useSidebarItemContext();
     const theme = useTheme().theme.sidebar.item;
 
-    const ListItem: FC<PropsWithChildren> = ({ children: wrapperChildren }) => (
-      <li>
-        {isCollapsed ? (
-          <Tooltip content={<TooltipContent>{children}</TooltipContent>} placement="right">
-            {wrapperChildren}
-          </Tooltip>
-        ) : (
-          wrapperChildren
-        )}
-      </li>
-    );
-
-    const TooltipContent: FC<PropsWithChildren> = ({ children }) => <Children>{children}</Children>;
-
-    const Children: FC<PropsWithChildren> = ({ children }) => (
-      <span
-        className={classNames(theme.content.base)}
-        data-testid="flowbite-sidebar-item-content"
-        id={`flowbite-sidebar-item-${id}`}
-      >
-        {children}
-      </span>
-    );
-
     return (
-      <ListItem>
+      <ListItem id={id} isCollapsed={isCollapsed} contentChildren={children}>
```

Comment:
I felt like having the same `id` there and at line 144 will be a problem. I mean, the `id` should be always unique. Right? Maybe adding some suffix to the one at 144 line, something like `${id}-children`.